packages/utils/src/validation/testValues.ts

```diff
@@ -42,18 +42,13 @@ export const invalidNames: Array<string> = [
 export const validEmails: Array<string> = [
 	'sarah@something',
 	'sarah@something.fr',
+	'sarah.connor@trmn.com',
 ];
 
-export const invalidEmails: Array<string> = [
-	'john',
-	'john@',
-	'john @',
-];
+export const invalidEmails: Array<string> = ['john', 'john@', 'john @', 'john\\@re'];
```

Comment:
When a user provides values like `john\@re`, it would be provided from email input as `john\\@re`